### PR TITLE
windows: Make mpconfigport.h up-to-date with the unix port

### PR DESCRIPTION
**Pull request by stinos**

### CODE REVIEW

File: windows/mpconfigport.h [hunk 1]
```diff
@@ -45,6 +45,8 @@
 #define MICROPY_DEBUG_PRINTERS      (1)
 #define MICROPY_USE_READLINE_HISTORY (1)
 #define MICROPY_HELPER_REPL         (1)
+#define MICROPY_REPL_EMACS_KEYS     (1)
```
Comment (pfalcon) [inline windows/mpconfigport.h]: Hm, strange, https://github.com/micropython/micropython/commit/7d588b0c7c9698d23b1e5e7b54e0e2094ad8f464 says "Disabled by default", but actually enables it. I guess that's what needs fixing. @pramasoul: FYI.

**Comment by pramasoul** [comment]
dpgeorge authored d8a7f8bf on Jul 26

**Comment by pfalcon** [comment]
> dpgeorge authored d8a7f8b on Jul 26

Ok, then we have different understanding of "disable by default". In the initial discussion, I meant to have Emacs keys support merged, but not enabled at all for any port - only be available there for folks who need them, to enable them in local builds.

@dpgeorge : Was enabling them by default after was the intention?

**Comment by dpgeorge** [comment]
> @dpgeorge : Was enabling them by default after was the intention?

"Disabling by default" to me means that in py/mpconfig.h it's disabled.  Then enabled on a per-port basis when desired.

I enabled emacs keys in stmhal because we have the code space and it makes the user experience better.  Same reason for unix (and then also allows to test this feature).

**Comment by pfalcon** [comment]
Ok, merged then, thanks.

**Pull request closed without merging**